### PR TITLE
Feature/add timezone frontend

### PR DESCRIPTION
**Pull request by alalonde**
Companion for #230

### CODE REVIEW

**Review by hammadj — COMMENTED** [review]
Needs to be rebased on master now that backend code is there. I saw that you made a PR into spacetime-informal to add typedefs, good stuff for taking the initiative! However, the type definitions arent actually showing up in the published NPM repo since the types folder isn't included in the package.json: https://github.com/spencermountain/spacetime-informal/blob/37015faffbd87d7ec30d477824cd6faf186bee8f/package.json#L26

As a result, there will also be a type error in the PlayerTimeZone.tsx file until that's resolved.

File: packages/web/components/Player/PlayerTimeZone.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      const {offset} = a.timezone().current;
+      if (offset > 0) {
+        offsetLabel = `(GMT +${offset})`;
+      } else if (offset < 0) {
+        offsetLabel = `(GMT ${offset})`;
+      }
+    } else {
+      tzLabel = player.tz;
+    }
+  }
```
Comment (hammadj) [inline packages/web/components/Player/PlayerTimeZone.tsx:31]: Should extract this logic into a util function, maybe in `utils/dateHelpers.ts` or something.

File: packages/web/components/Player/PlayerTimeZone.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+type Props = {
+  player: PlayerFragmentFragment;
+};
+
+export const PlayerTimeZone: React.FC<Props> = ({ player }) => {
+  let tzLabel = '-';
+  let offsetLabel = null;
+  if (player.tz) {
+    const a = spacetime.now().goto(player.tz)
```
Comment (hammadj) [inline packages/web/components/Player/PlayerTimeZone.tsx:16]: what is "a" supposed to be? A more descriptive name would make it more readable.

File: packages/web/components/Player/PlayerTimeZone.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import { display } from 'spacetime-informal';
+
+type Props = {
+  player: PlayerFragmentFragment;
+};
+
+export const PlayerTimeZone: React.FC<Props> = ({ player }) => {
+  let tzLabel = '-';
+  let offsetLabel = null;
+  if (player.tz) {
```
Comment (pakokrew) [inline packages/web/components/Player/PlayerTimeZone.tsx:15]: `useMemo` so it doesn't get recomputed at each redraw

File: packages/web/components/Setup/SetupSkills.tsx [hunk 4]
```diff
@@ -20,6 +22,31 @@ export const SetupSkills: React.FC = () => {
...
   } = useSetupFlow();
+  const { user } = useUser({ redirectTo: '/' });
+  const toast = useToast();
+  
+  const [updateSkillsRes, updateSkills] = useUpdatePlayerSkillsMutation();
+
+  const handleNextPress = async () => {
+    if (!user) return;
+
+    const { error } = await updateSkills({
```
Comment (pakokrew) [inline packages/web/components/Setup/SetupSkills.tsx:33]: thank you 😁

**Review by hammadj — APPROVED** [review]

**Pull request merged**